src/js/admin/screens/groupsScreen.js

```diff
@@ -0,0 +1,150 @@
+const tableHeader = `
+  <div class="tr groups-table-header">
+    <div class="td sr-img">Image</div>
+    <div class="td sr-number">Number</div>
+    <div class="td sr-id">Key</div>
+    <div class="td sr-name">Name</div>
+    <div class="td sr-edit">Edit</div>
+  </div>
+`;
+
+App.renderGroupsScreen = () => {
+    const groupKeys = Object.keys(App.groups);
```

Comment:
mel bys dodrzet spolecnou konvenci psani kodu ve ktere pisu ja, konkretne tady mas indentaci 4 mezery a ja mam 2 mezery, zkus si to nastavit abys mel automaticke zarovnani v editoru